bigtable/google/cloud/bigtable/table.py

```diff
@@ -462,11 +483,15 @@ def _do_mutate_retryable_rows(self):
                 if entry.status.code == 0:
                     self.rows[index].clear()
 
-        assert len(retryable_rows) == num_responses
+        if len(retryable_rows) != num_responses:
+            raise RuntimeError(
+                'Unexpected the number of responses', num_responses,
```

Comment:
Did you mean to include the `the` here?

Reply:
I did not, good catch.

Reply:
@justinseinlin To avoid making CI restart, you can include this change in your PR? Or I can do once I get an LGTM from a reviewer?

Reply:
I'll include this in my PR.